extensions/track-label/functions/src/shipengine-sdk.ts

```diff
@@ -0,0 +1,83 @@
+import axios, { AxiosInstance, AxiosRequestConfig } from 'axios';
+import ShipEngine from 'shipengine';
+import { camelizeKeys } from 'humps';
+
+// import config from './config';
+import {
+  CreateWebhookParams,
+  CreateWebhookResponse,
+  ListWebhooksResponse,
+  Webhook,
+} from './types';
+interface TrackLabelParams {
+  carrierCode: string;
+  trackingNumber: string;
+}
+export class ShipEngineSDK extends ShipEngine {
+  private client: AxiosInstance;
+  constructor(
+    apiKey: string,
+    clientConfig: AxiosRequestConfig = {
+      baseURL: 'https://api.shipengine.com',
+      headers: {
+        Host: 'api.shipengine.com',
+        'API-Key': apiKey,
+        'Content-Type': 'application/json',
+      },
+    }
+  ) {
+    super(apiKey);
+    this.client = axios.create(clientConfig);
+  }
```

Comment:
I'm *pretty* sure you can add functions to a JS class prototype so you don't have to fully extend the ShipEngine client and re-define the whole constructor. I haven't worked with JS classes much in a while, but I think it should be doable like this:

```
import ShipEngine from 'shipengine';

ShipEngine.prototype.startTrackingLabel = async (params: TrackLabelParams): Promise<void> => {
  ...
};

export { ShipEngine };
```